docker-compose.yml

```diff
@@ -38,18 +38,21 @@ services:
     ports:
       - "80:8980"
     depends_on:
-      - app
+      - essi_app
       - worker
 
-  app:
+  essi_app:
```

Comment:
this feels like an arbitrary change?

Reply:
I see - this is about google having app as a domain. maybe just "essi" is a better name?